src/Instagram.php

```diff
@@ -76,9 +76,19 @@ public function refreshToken($token)
         return $this->http->get($url);
     }
 
-    public function fetchMedia(AccessToken $token, $limit = 20)
+    public function fetchMedia(AccessToken $token, $limit = null)
```

Comment:
still honours the default 20 from `$profile->feed()` or `$profile->refreshFeed()` in Instagram.php 

unless `null` is passed through